tools/build_cdn.js

```diff
@@ -0,0 +1,90 @@
+const fs = require("fs").promises;
+const glob = require("glob");
+const zlib = require('zlib');
+const { getLanguages } = require("./lib/language");
+const { filter } = require("./lib/dependencies");
+const config = require("./build_config");
+const { install, install_cleancss, mkdir } = require("./lib/makestuff");
+const log = (...args) => console.log(...args);
+const { buildBrowserHighlightJS } = require("./build_browser");
+const path = require("path");
+
+async function buildCDN(options) {
+  install("./LICENSE", "LICENSE");
+  install("./README.md","README.md");
+
+  installStyles();
+
+  // all the languages are built for the CDN and placed into `/languages`
+  const languages = await getLanguages();
+  await installLanguages(languages);
+
+  // filter languages for inclusion in the highlight.js bundle
+  const embedLanguages = filter(languages, options["languages"])
+
+  var size = await buildBrowserHighlightJS(embedLanguages, {minify: options.minify})
+
+  log("-----")
+  log("Embedded Lang       :",
+    embedLanguages.map((el) => el.minified.length).reduce((acc, curr) => acc + curr, 0), "bytes");
+  log("All Lang            :",
+    languages.map((el) => el.minified.length).reduce((acc, curr) => acc + curr, 0), "bytes");
+  log("highlight.js        :",
+    size.regular, "bytes");
+
+  if (options.minify) {
+    log("highlight.min.js    :", size.minified ,"bytes");
+    log("highlight.min.js.gz :", zlib.gzipSync(size.data).length ,"bytes");
+  } else {
+    log("highlight.js.gz     :", zlib.gzipSync(size.fullSrc).length ,"bytes");
```

Comment:
`fullSrc` -> `data`

Reply:
Not they are different things, but the naming is poor.  I will improve naming of these items.

Reply:
Yeah, I see now. Previously I got an error during CDN build with --no-minify option in this line, so I thought that the name was incorrect. Can't reproduce now.